rosbridge_library/src/rosbridge_library/internal/publishers.py

```diff
@@ -223,13 +223,13 @@ def register(self, client_id, topic, node_handle, msg_type=None, latch=False, qu
             self._publishers[topic] = MultiPublisher(
                 topic, node_handle, msg_type=msg_type, latched_client_id=latched_client_id, queue_size=queue_size)
         elif latch and self._publishers[topic].latched_client_id != client_id:
-            node_handle.get_logger().warn("Client ID %s attempted to register topic [%s] as latched "
-                    "but this topic was previously registered." % (client_id, topic))
+            node_handle.get_logger().warn(f"Client ID {client_id} attempted to register topic [{topic}] as "
+                    "latched but this topic was previously registered.")
             node_handle.get_logger().warn("Only a single registered latched publisher is supported at the time")
         elif not latch and self._publishers[topic].latched_client_id:
-            node_handle.get_logger().warn("New non-latched publisher registration for topic [%s] which is "
+            node_handle.get_logger().warn(f"New non-latched publisher registration for topic [{topic}] which is "
                     "already registered as latched. but this topic was previously "
-                    "registered." % topic)
+                    "registered.")
```

Comment:
nit: this could be merged with the line above now

doesn't matter too much though, I assume we will run a formatter over all this code soon